src/stories/ThreePointTopologies.stories.tsx

```diff
@@ -0,0 +1,209 @@
+import type { Meta, StoryObj } from "storybook-solidjs";
+import { Bluefish } from "../bluefish";
+import withBluefish, { WithBluefishProps } from "../withBluefish";
+import { Show, For, ParentProps, mergeProps, splitProps } from "solid-js";
+import Group from "../group";
+import Text from "../text";
+import { StackH } from "../stackh";
+import Background from "../background";
+import Circle from "../circle";
+import Ref from "../ref";
+import { StackV } from "../stackv";
+import Rect from "../rect";
+import { createName } from "../createName";
+import Layout from "../layout";
+import { PaperScope } from "paper/dist/paper-core";
+import { BBox, Transform } from "../scenegraph";
+import { maybeSub } from "../util/maybe";
+
+const meta: Meta = {
+  title: "Example/ThreePointTopologies",
+};
+
+export default meta;
+type Story = StoryObj;
+
+const Path = withBluefish(
+  (rawProps: WithBluefishProps<{ d: string; x?: number; y?: number }>) => {
+    const props = mergeProps(
+      {
+        "stroke-width": 3,
+        stroke: "black",
+        position: "relative",
+        fill: "none",
+        d: "",
+        x: undefined,
```

Comment:
Are these `undefined` merges necessary?

Reply:
No they are not, I had them to reference before I added Types. Removing.